spec/lib/gitlab/oauth/auth_hash_spec.rb

```diff
@@ -3,53 +3,97 @@
 describe Gitlab::OAuth::AuthHash do
   let(:auth_hash) do
     Gitlab::OAuth::AuthHash.new(double({
-      provider: 'twitter',
-      uid: uid,
+      provider: provider_ascii,
```

Comment:
Use 2 spaces for indentation in a hash, relative to the first position after the preceding left parenthesis.